bindings/cs/MongoDB.Libmongocrypt/Library.cs

```diff
@@ -512,6 +515,10 @@ internal class Delegates
             /// uint64_t mongocrypt_crypt_shared_lib_version(const mongocrypt_t* crypt);
             /// </summary>
             public delegate ulong mongocrypt_crypt_shared_lib_version(MongoCryptSafeHandle handle);
+            /// <summary>
+            /// bool mongocrypt_setopt_fle2v2(mongocrypt_t *crypt, bool enable)
+            /// </summary>
+            public delegate bool mongocrypt_setopt_fle2v2(MongoCryptSafeHandle handle, bool enable);
```

Comment:
I think you need to add this attribute: `[return: MarshalAs(UnmanagedType.I1)]`

Reply:
Done.